src/client/components/LeftBlueBar/LeftBlueBar.styles.css

```diff
@@ -63,6 +66,13 @@
   font-family: 'Chakra Petch', sans-serif;
 }
 
+.google_logo {
+  width: 1.3rem;
+  height: 1.3rem;
```

Comment:
Remove one, either `width `or `height`. It is better to just provide one to avoid distorting the images